app/src/release/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugNotificationManager.kt

```diff
@@ -0,0 +1,9 @@
+package co.netguru.baby.monitor.client.feature.debug
+
+import android.content.Context
+import javax.inject.Inject
+
+class DebugNotificationManager @Inject constructor() {
+    fun show(context: Context) {}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/a089e65afe6044fe8b2c4847ffd33998441cf079/app/src/release/kotlin/co/netguru/baby/monitor/client/feature/debug/DebugNotificationManager.kt#L7"></span>Function parameter context is unused.</td>
    </tr>
  </tbody>
</table>